lilach-client/src/main/resources/org/cshaifasweng/winter/Empty_Cart_Guest.fxml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is nice, but try to reduce the prefHeight of the anchor pane a bit, or maybe just use the computed size there.